src/decirest.erl

```diff
@@ -94,8 +94,8 @@ child_urls_map([Child | Children], Req, State, Map) ->
     {true, _, _} ->
       child_urls_map(Children, Req, State, Map)
   end;
-child_urls_map([], _Req, _State, Map) ->
-  Map.
+child_urls_map([], #{path := Path} = Req, _State, Map) ->
+  Map#{<<"doc_url">> => pretty_path([<<"/apidoc/path">>, Path])}.
```

Comment:
this is ok for now, but shouldn't we make this configurable. If I remember correctly we extracted the actual apidoc out from decirest so that it was owned by the application using decirest. If so we should fetch it from a callback from that module or similar.

Reply:
yes me and Jonathan talked about maybe adding more of the general doc stuff to decirest.